api/models.py

```diff
@@ -89,3 +90,104 @@ def save(self, *args, **kwargs):
     def __str__(self):
         return '{}{}{}'.format(self.item_code, self.serial_number,
                                self.model_number)
+
+
+class UserManager(BaseUserManager):
+    def create_user(self, email, name, cohort, slack_handle,
+                    is_active=True, is_admin=False, is_staff=False,
+                    phone_number=None, picture=None, password=None):
+        if not email:
+            raise ValueError("Email address is required")
+        elif not name:
+            raise ValueError("Name is required")
+        elif not cohort:
+            raise ValueError("Cohort is required")
+        elif not slack_handle:
+            raise ValueError("Slack handle is required")
+
+        user = self.model(
+            email=self.normalize_email(email),
+            name=name,
+            cohort=cohort,
+            slack_handle=slack_handle,
+        )
+
+        user.set_password(password)
+        user.picture = picture
+        user.staff = is_staff
+        user.admin = is_admin
+        user.active = is_active
+        user.phone_number = phone_number
```

Comment:
we can also pass these fields to `self.model` since they're defined on the models level. and if we pass them as kwargs it'd make it easier. (see [here](https://github.com/django/django/blob/master/django/contrib/auth/models.py#L142))